walletnode/examples/configs/localnet.yml

```diff
@@ -1,6 +1,6 @@
 pastel-api:
   hostname: "127.0.0.1"
-  port: 12170
+  port: 29932
```

Comment:
Don't change configuration on master that was tuned to match with mock-network.

Reply:
changed to the old one
thank you